pages/review.tsx

```diff
@@ -129,17 +129,30 @@ function ApprovalList({ type, category }: { type: RequestType; category: ReviewF
       {requests.map((requestObj: any) => (
         <Box sx={{ px: 3 }} key={requestObj._id}>
           <Grid container spacing={1} sx={requestObj.approvalType === 'Manager' ? managerStyling : reviewerStyling}>
-            <Grid item xs={12} md={6} lg={7}>
+            <Grid item xs={12} md={9}>
```

Comment:
I would change this back to `<Grid item xs={12} md={6} lg={7}>` now that we're using the chip